mender-deb-package

```diff
@@ -64,7 +57,7 @@ Build-Depends: debhelper (>= 9)
 
 Package: mender
 Architecture: armhf
-Depends: \${shlibs:Depends}, \${misc:Depends}
+Depends: libc6 (>= 2.12), liblzma5 (>= 5.1.1)
```

Comment:
I have set these dependencies based on what Raspbian upstream has today as latest.

However, we are building with libc6 2.27, and 5.2.4. Will that be a problem?

It works on my Raspberry Pi 3...

Reply:
I couldn't find any specific information about versioning of either component, but I think libc is extremely stable, and it looks like liblzma is using semantic versioning, so we can at least drop the last component (patch versions should be both forwards and backwards compatible).

But this is one case where I would consider not using the latest debian to compile, but rather an older one. That might land you on older versions of both of them.

Reply:
Mmmm. Good idea, so using an older build machine will ensure that the latest Raspbian (upstream) has newer versions of the dependencies. I will fix it to build from an older build host

Did I understand correctly?

And yes, I will drop the patch part on the versioning.

Reply:
As discussed in person; this conversation is really a misunderstanding. The "build" versions are not from Debian upstream, but from the downloaded bootlin toolchain (libc6) or the dependencies source code (liblzma).

Ironically, this PR started with the intention of aligning mender-convert and the .deb package support wrt ARM. Hence, we should not change here the toolchain nor liblzma source we are using to compile.

Bottom line: we know both dependencies are very stable and we know "it works". So we will leave it as it is. Objections?

Reply:
:+1: